imap_processing/tests/glows/test_glows_l1b_data.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import pytest
+
+from imap_processing.glows.l1b.glows_l1b_data import AncillaryParameters, DirectEventL1B
+
+
+def test_glows_l1b_ancillary_file():
+    fake_good_input = {
```

Comment:
Is this the same as your ancillary data fixture?

Reply:
This is the same format as the ancillary data file with different input values.